### PR TITLE
Add detection for .yaml and .yml

### LINKED ISSUE #706: Support .taprc{.yml,.yaml}
A little quality-of-life request: support `.taprc.yml` and/or `.taprc.yaml` file names. This would allow editors to recognize the format of the file and behave accordingly.

### PR DESCRIPTION
**Pull request by jsumners**
This resolves #706.

I based my tests on `test/run/bad-rcfile.js` because I didn't see any other tests that cover parsing of rc files that I could learn from.

I am not clear how to run the tests and generate an HTML coverage report so that I can see what branch(es) are not being covered. When I run `npm run unit` I can see that some branch coverage is missing, but not what it is.

### CODE REVIEW

File: bin/run.js [hunk 1]
```diff
@@ -805,7 +805,20 @@ const parseRcFile = path => {
   let contents
   try {
     contents = fs.readFileSync(path, 'utf8')
-  } catch (er) {
+  } catch {
+    try {
+      if (fs.accessSync(path + '.yaml') === undefined) {
```
Comment (isaacs) [inline bin/run.js]: Why the access checks?  Just read the file, if it throws, it's not there or you can't read it.

Comment (jsumners) [reply bin/run.js]: Okay. I have updated the PR accordingly.

**Comment by isaacs** [comment]
Landed on v15, will be in next release.  Thanks!

**Comment by jsumners** [comment]
image

**Pull request closed without merging**